main.go

```diff
@@ -118,6 +119,13 @@ func main() {
 	c, cleanup := NewCollection(outputFile)
 	defer cleanup()
 
+	if noDetailedCollection {
+		c.Detailed = false
+		c.Log.Warn("Detailed collection is disabled")
```

Comment:
Maybe a comment about "detailed collection is and means"

Reply:
Done with 8b74214.